tools/db_stress.cc

```diff
@@ -1620,6 +1643,47 @@ class StressTest {
       }
     }
 
+#ifndef ROCKSDB_LITE
+    if (FLAGS_enable_secondary) {
+      now = FLAGS_env->NowMicros();
+      fprintf(stdout, "%s Start to verify secondaries against primary\n",
+              FLAGS_env->TimeToString(static_cast<uint64_t>(now) / 1000000)
+                  .c_str());
+    }
+    for (size_t k = 0; k != secondaries_.size(); ++k) {
```

Comment:
do we want to execute this for loop when `FLAGS_enable_secondary == false`?

Reply:
If `FLAGS_enable_secondary` is false, then `secondaries_` will be empty, thus the loop won't have any effect.